hydro_lang/Cargo.toml

```diff
@@ -41,6 +41,7 @@ tokio = { version = "1.29.0", features = [ "full" ] }
 toml = { version = "0.8.0", optional = true }
 trybuild-internals-api = { version = "1.0.99", optional = true }
 ctor = "0.2"
+rand = "0.8.5"
```

Comment:
This shouldn't be here presumably

Reply:
It's used by one of the non-determinism examples I believe

Reply:
(ought to be in dev-dependencies if that example is made, I think)